### PR TITLE
add document id back into tsv

### PR DESCRIPTION
**Pull request by simonjupp**
No real need for review, just showing you where the code is for adding/removing default columns

### CODE REVIEW

**Review by hannes-ucsc — COMMENTED** [review]
It's not apparent to me how this adds document_id back to the TSV. It just seems to fix a bunch of invalid regexes?

**Comment by hannes-ucsc** [comment]
@GPelayo please track this PR and keep your copy of the script in sync.

File: hca_bundle_tools/file_metadata_to_csv.py [hunk 1]
```diff
@@ -24,19 +24,20 @@ def __init__(self, order = None, ignore = None, format_filter = None):
...
+             "^specimen_from_organism.*",
+             "^cell_suspension.*",
+             "^.*protocol.*",
              "^project."
         ]
 
         self.default_ignore = ignore if ignore else [
             "describedBy",
             "schema_type",
-            "provenance",
```
Comment (simonjupp) [inline hca_bundle_tools/file_metadata_to_csv.py:39]: @hannes-ucsc by default anything in the `provenance` block is ignored when we build the TSV (including the document id). I changed it so only the `submission_date` and `update_date` are ignore now.

**Comment by simonjupp** [comment]
> @GPelayo please track this PR and keep your copy of the script in sync.

Why copy of the script? Try not to fork it if possible. If you have any modifications please push back.

File: hca_bundle_tools/file_metadata_to_csv.py (hunk 1 above)
Comment (hannes-ucsc) [reply hca_bundle_tools/file_metadata_to_csv.py:39]: I usually keep unrelated changes in separate commits.

Comment (simonjupp) [reply hca_bundle_tools/file_metadata_to_csv.py:39]: not sure how those regexes got screwed up in the commit. Think my IDE did something!

**Comment by hannes-ucsc** [comment]
@simonjupp 

> Why copy of the script? Try not to fork it if possible. If you have any modifications please push 
back.

We'll do just that: 

https://github.com/HumanCellAtlas/hca_bundles_to_csv/issues/4

**Pull request merged**